lib/Command/TestPush.php

```diff
@@ -62,32 +68,53 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 		}
 
 		$userId = $input->getArgument('user-id');
-		$subject = 'Testing push notifications';
-
 		$user = $this->userManager->get($userId);
 		if (!$user instanceof IUser) {
-			$output->writeln('Unknown user');
+			$output->writeln('<error>Unknown user</error>');
 			return 1;
 		}
 
+		if ($input->getOption('talk')) {
+			$failed = $this->sendNotification($output, $user, 'talk');
+		} else {
+			$failed = false;
```

Comment:
Fine by me, although it feels like this should be init above, because when no option is set, `$failed` does not have a value in Line 85-88.

Reply:
When option talk is NOT set, this line here will init it?
When option talk is set, two lines above will

I have it outside before followed by ` || $failed` in the diff, but psalm complained that it was always falsy.
But removing ` || $failed` meant that when someone reorders the entries, it could break by accident, so I thought this way it's clearer that you have to check what you do if you change/reorder

Reply:
> When option talk is NOT set, this line here will init it?

Right, sorry, mixed two replies into one and the last part is indeed wrong. But yea, as said, fine by me this way.